go/flags/endtoend/vtgr.txt

```diff
@@ -62,8 +62,6 @@ Usage of vtgr:
       --scan_repair_timeout duration                                     Time to wait for a Diagnose and repair operation. (default 3s)
       --security_policy string                                           the name of a registered security policy to use for controlling access to URLs - empty means allow all for anyone (built-in policies: deny-all, read-only)
       --service_map StringList                                           comma separated list of services to enable (or disable if prefixed with '-') Example: grpc-queryservice
-      --sql-max-length-errors int                                        truncate queries in error logs to the given length (default unlimited)
```

Comment:
fwiw, i always[^1] run this command anytime i do a flags PR:

```
$ find go/flags/endtoend -iname '*.txt' | xargs basename | cut -d. -f1 | xargs -I{} bash -c "{} --help 2>go/flags/endtoend/{}.txt"
```

[^1]: okay very occasionally i forget

Reply:
I ran the command and there are no changes. `TestHelpOutput` also passes 👍. Merging the PR now.